client/scripts/views/components/component_kit/cw_text.tsx

```diff
@@ -0,0 +1,52 @@
+/* @jsx m */
+
+import m from 'mithril';
+
+import 'components/component_kit/cw_text.scss';
+
+import { ComponentType } from './types';
+
+type FontStyle = 'regular' | 'semi-bold' | 'bold' | 'black' | 'italic';
+type FontType = 'd1' | 'd2' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'b1' | 'b2';
+
+type TextAttrs = {
+  disabled?: boolean;
+  fontStyle: FontStyle;
+  noWrap?: boolean;
+  type: FontType;
+};
+
+export const getTextClasses = (
```

Comment:
why do we have this method in so many class components? can't it just be written once?